sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -597,7 +597,7 @@ public virtual async Task<Response<ConfigurationSetting>> GetAsync(string key, s
         /// <param name="acceptDateTime">The setting will be retrieved exactly as it existed at the provided time.</param>
         /// <param name="requestOptions"></param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        public virtual Response<ConfigurationSetting> Get(string key, string label, DateTimeOffset acceptDateTime, ConditionalRequestOptions requestOptions, CancellationToken cancellationToken = default)
```

Comment:
We should either remove both methods and tests or keep tests.

Reply:
Our member interceptors don't handle internal methods. Do you want me to add support for them?

Reply:
It should be as easy as adding InternalsVisibleTo, see https://github.com/Azure/azure-sdk-for-net/pull/7843 <- this change

Reply:
`InternalsVisibleTo` is already there (https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/src/Properties/AssemblyInfo.cs#L6), but it doesn't make internal methods public, it changes bypassing rules. 
Because of that, code in interceptors that relies on reflection, e.g.:
```cs
        private static MethodInfo GetMethod(IInvocation invocation, string nonAsyncMethodName, Type[] types)
        {
            return invocation.TargetType.GetMethod(nonAsyncMethodName, BindingFlags.Public | BindingFlags.Instance, null, types, null);
        }
```
can't find required method.

Reply:
Yes, you are right, I would change it to allow proxying to internal methods but only if the method that is being called is also internal.

Reply:
But only if we care about keeping these methods around.

Reply:
Also, if anyone is interested how you can bypass the rules from the calling assembly:
https://www.strathweb.com/2018/10/no-internalvisibleto-no-problem-bypassing-c-visibility-rules-with-roslyn/

Reply:
I like having these functions around in case we ever decide to expose them.  But, if it's too much work, I'm ok removing them.  If we do that, we'll need to rewrite the convenience methods to include that logic.

Reply:
Fixed